enterprise/server/cmd/ci_runner/main.go

```diff
@@ -667,6 +684,27 @@ func (ws *workspace) setup(ctx context.Context) error {
 	return ws.sync(ctx)
 }
 
+func (ws *workspace) applyPatch(ctx context.Context, digestString string, bsClient bspb.ByteStreamClient) error {
```

Comment:
uber uber nit but I think ordering like (ctx context.Context,  bsClient bspb.ByteStreamClient, digestString string) makes more sense (since digestString is the arg that changes most often)

Reply:
makes sense, done